### ISSUE TITLE
Practical fix for "503 blocked by Instagram"

### ISSUE THREAD

**Issue by kalak-sudo** [cloudrac3r/bibliogram#84]
Hi!

I am having this issue with my new server, and as I have read the docs, I am trying to figure out how to fix it. 

First attempt:

config.js with the following content:

module.exports = {
    website_origin: "https://bibliogram.MYSERVER.COM",
    tor: {
        enabled: true
    }
as_assistant: {
        enabled: true,
        require_key: true,
        keys: [
                "first_key",
                "second_key"
        ]
}
}

And I get "**Bad Gateway**" now. What are the practical steps I can follow to solve this issue @cloudrac3r ? 

I have read: 

[https://github.com/cloudrac3r/bibliogram/wiki/Rate-limits](https://github.com/cloudrac3r/bibliogram/wiki/Rate-limits)

and

[https://github.com/cloudrac3r/bibliogram/wiki/Assistants](https://github.com/cloudrac3r/bibliogram/wiki/Assistants)

Thanks!

**Comment by cloudrac3r** [comment]
Duplicate of #77?

**Comment by kalak-sudo** [comment]
Yes, sorry. I looked for similar issues before opening a new one, but didn't find #77 . You can close or delete this issue @cloudrac3r  :)
